pkgs/applications/terminal-emulators/kgx/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, stdenv
+, genericBranding ? false
+, fetchgit
+, gettext
+, gnome3
+, gtk3
+, libhandy
+, pcre2
+, vte
+, appstream-glib
+, desktop-file-utils
+, git
+, meson
+, ninja
+, pkg-config
+, python3
+, sassc
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation {
+  pname = "kgx";
+  version = "unstable-2021-03-12";
+
+  src = fetchgit {
```

Comment:
It would be nice if fetchFromGitLab was used.

Reply:
Ah cool, just checked and I didn't know `fetchFromGitLab` supported self-hosted instances.